### PR TITLE
Add timezone fixes & expectedTimezone getter

### PR DESCRIPTION
**Pull request by kellyje**

### CODE REVIEW

File: formatters.js [hunk 1]
```diff
@@ -59,12 +59,13 @@ DateTools.fuzzyDay = function (date) {
     TimeTracker.changeAt(midnightTomorrow.toDate());
   }
 
-  var dayMoment = moment(date).startOf('day');
-  var todayMoment = moment().startOf('day');
+  var timezone = DateTools.getExpectedTimezone();
+  var dayMoment = moment.tz(date, timezone).startOf('day');
```
Comment (jperl) [inline formatters.js]: Will this work when the expected timezone is null?

Seems like some tests would be nice.

Comment (jperl) [inline formatters.js]: I'll add the tests.

Comment (kellyje) [inline formatters.js]: There should definitely be a check around it. I'll add it in a bit. In our use cases though, it shouldn't be null.

Comment (jperl) [inline formatters.js]: I'll fix it. I am fixing the tests now. Many tests were not passing :(

**Pull request merged**